modules/ROOT/partials/configuration/content_css_cors.adoc

```diff
@@ -1,3 +1,4 @@
+[[content_css_cors]]
 == `content_css_cors`
```

Comment:
At the end of this file, there a code block that needs converting to the new syntax `[source, js]`